src/internal/testing/TestScheduler.ts

```diff
@@ -409,14 +413,62 @@ export class TestScheduler extends VirtualTimeScheduler {
     this.maxFrames = Infinity;
     this.runMode = true;
     AsyncScheduler.delegate = this;
+    dateTimestampProvider.delegate = this;
+    performanceTimestampProvider.delegate = this;
 
-    const helpers = {
+    let animationFramesHandle = 0;
+    let animationFramesQueue: Map<number, FrameRequestCallback> | undefined;
+
```

Comment:
Be sure to add comments as to the "why" around here. We don't do that enough in our library, I think, and while it might be obvious to us right now, maybe it won't be 5 years from now.